### ISSUE TITLE
Some column names are skipped by annotate_fibermap()

### ISSUE THREAD

**Issue by geordie666** [desihub/desispec#2218]
Units are skipped for 5 columns that are not covered by `desispec.io.fibermap.annotate_fibermap()`:

```
MEAN_MJD
MIN_MJD
MAX_MJD
FIRSTNIGHT
LASTNIGHT
```
See https://github.com/desihub/redrock/pull/292.

Add these units.

**Comment by weaverba137** [comment]
@sbailey, I'm a bit puzzled by this. In fact, those columns *are* described by `desispec.io.fibermap.fibermap_columns`. Has redrock simply not picked up the revisions to `annotate_fibermap`?

**Comment by weaverba137** [comment]
@geordie666, was this meant to be closed by #2219? But I'd like to hear a bit more about why this was necessary.  I thought that the work @sbailey & I did on updating the fibermap columns (#2176) was comprehensive.

**Comment by sbailey** [comment]
@weaverba137 yes I think this is a dangling ticket that should have been closed after #2219 was merged.  I also thought that #2176 was the comprehensive list of columns but we missed those 5 columns.  I think the reason is that they were added after Iron was run but I was basing my column checking on Iron and forgot those post-Iron additions.  I'll close this ticket.

**Comment by geordie666** [comment]
@weaverba137: Yes, sorry, this issue should have been closed by #2219. 

As I recall, @sbailey thought your work was comprehensive, too. But, I had to add the five columns outlined above to `desispec.io.fibermap.fibermap_columns` as part of this PR:

https://github.com/desihub/desispec/blame/main/py/desispec/io/fibermap.py#L143-L147

I'm not sure why these columns were missed, it was likely just an oversight. Note that I also added the 5 missing columns to the data model:

https://github.com/desihub/desidatamodel/pull/192
